lib/elixir/lib/module.ex

```diff
@@ -879,8 +910,8 @@ defmodule Module do
       end
 
   """
-  @spec get_attribute(module, atom) :: term
-  def get_attribute(module, key) do
+  @spec get_attribute(module, key :: atom) :: (value :: term)
```

Comment:
I don't think we need to name the types here (`key ::` and `value ::`), they're clear from the function arguments and the function docs.